src/containers/DateTime.js

```diff
@@ -30,23 +30,54 @@ The default format is `lll` -> `Aug 22, 2020 1:13 PM`
 */
 
 export function DateTime(props) {
-
+	const [locale, setLocale] = useState('');
 	const language = useSelector(state => state.language.language)
+	
+	useEffect(() => {
+		if (language && language !== "en") {
+			fetchLocale()
+		} else {
+			setLocale(undefined);
+		}
+	}, [language])
+
+	const fetchLocale = async () => {
+		// Don't remove comment inside import
+		// it's webpack's dynamic expression
+		const importedLocale = await import(
+			/* webpackMode: "lazy", webpackChunkName: "df-[index]" */
+			`date-fns/locale/${language}/index.js`
+		);
+		setLocale(importedLocale.default);
+	}
 
 	if ((props.value === null) || (props.value === undefined)) {
 		return (
 			<span className="datetime">{' '}</span>
-		)
+		);
 	}
 
-	const m = isNaN(props.value) || props.value > 9999999999 ? moment(props.value) : moment(props.value * 1000);
-	m.locale(language.slice(0, 2));
+	if (new Date(props.value).toString() === "Invalid Date") {
+		return (
+			<span className='datetime'>
+				<i className="cil-clock pr-1"></i>
+				Invalid Date
```

Comment:
@titkov49 Is it possible to translate it?

Reply:
> @titkov49 Is it possible to translate it?

@Pe5h4 I can edit it as common translation:
```js
t("Invalid Date")
```